Resources/Prototypes/Entities/Structures/Piping/Disposal/pipes.yml

```
@@ -8,6 +8,8 @@
     snap:
     - Disposal
   components:
+  - type: Sprite
+    netsync: false
```

```suggestion
  # - type: Sprite
  #   netsync: false
```

Too based at this time.